### PR TITLE
Release/2.1.5

### PR DESCRIPTION
**Pull request by HanlinMiao**
New PR for Release/2.1.5

### CODE REVIEW

File: nautobot/docs/release-notes/version-2.1.md [hunk 1]
```diff
@@ -72,6 +72,40 @@ Support for versions of PostgreSQL prior to 12.0 has been removed as these versi
 Support for `HIDE_RESTRICTED_UI` has been removed. UI elements requiring specific permissions will now always be hidden from users lacking those permissions. Additionally, users not logged in will now be automatically redirected to the login page.
 
 <!-- towncrier release notes start -->
+## v2.1.5 (2024-02-20)
```
Comment (gsnider2195) [inline nautobot/docs/release-notes/version-2.1.md:75]:
```suggestion
## v2.1.5 (2024-02-21)
```

File: nautobot/docs/release-notes/version-2.1.md [hunk 2]
```diff
@@ -72,6 +72,40 @@ Support for versions of PostgreSQL prior to 12.0 has been removed as these versi
...
+- [#5210](https://github.com/nautobot/nautobot/issues/5210) - Added `METRICS_AUTHENTICATED` setting to control authentication for the HTTP endpoint `/metrics`.
+- [#5243](https://github.com/nautobot/nautobot/issues/5243) - Added support for setting display_field on DynamicModelChoiceField to nested values in suggested choices list.
+
+### Changed
+
+- [#5171](https://github.com/nautobot/nautobot/issues/5171) - Changed the tagging of `nautobot-dev` Docker images to reserve the `latest` and `latest-py<version>` tags for the latest stable release of Nautobot, rather than the latest build from the `develop` branch.
+- [#5254](https://github.com/nautobot/nautobot/issues/5254) - Changed `TreeQuerySet.ancestors` implementation to a more efficient approach for shallow trees.
+- [#5254](https://github.com/nautobot/nautobot/issues/5254) - Changed the location detail view not to annotate tree fields on its queries.
+- [#5267](https://github.com/nautobot/nautobot/issues/5267) - Updated navbar user dropdown with chevron.
+- [#5267](https://github.com/nautobot/nautobot/issues/5267) - Reorganized navbar css rules in `base.css`.
```
Comment (gsnider2195) [inline nautobot/docs/release-notes/version-2.1.md:93]: housekeeping

File: nautobot/docs/release-notes/version-2.1.md [hunk 3]
```diff
@@ -72,6 +72,40 @@ Support for versions of PostgreSQL prior to 12.0 has been removed as these versi
...
+
+### Fixed
+
+- [#5058](https://github.com/nautobot/nautobot/issues/5058) - Changed more filter parameters from location_id to location in virtualization/forms.py
```
Comment (gsnider2195) [inline nautobot/docs/release-notes/version-2.1.md:97]:
```suggestion
- [#5058](https://github.com/nautobot/nautobot/issues/5058) - Changed more filter parameters from `location_id` to `location` in `virtualization/forms.py`.
```

File: nautobot/docs/media/nautobot_application_stack_low_level.txt [hunk 4]
```diff
@@ -1 +1 @@
-https://www.canva.com/design/DAF0J8lu-9M/-X8Ke2VfMKG5ZAXKFsS_3A/edit
+https://www.canva.com/design/DAFzIltG0Ko/N4gMzo6qwylPRN0HAAfCIg/edit
```
Comment (gsnider2195) [inline nautobot/docs/media/nautobot_application_stack_low_level.txt:1]: This link needs to be changed to view only

**Review by gsnider2195 — APPROVED** [review]
Don't squash!

File: nautobot/docs/media/nautobot_application_stack_low_level.txt (hunk 4 above)
Comment (HanlinMiao) [reply nautobot/docs/media/nautobot_application_stack_low_level.txt]: Change the share permissions to "anyone with the link can view". Maybe other people should access it to see if they can edit it still.
Screen Shot 2024-02-21 at 11 36 00 AM

Comment (glennmatthews) [reply nautobot/docs/media/nautobot_application_stack_low_level.txt]: New link appears view-only to me.

**Pull request merged**